Core/src/Vertexing/AdaptiveGridTrackDensity.cpp

```diff
@@ -12,16 +12,39 @@
 
 #include <algorithm>
 
-float Acts::AdaptiveGridTrackDensity::getBinCenter(int bin, float binExtent) {
+namespace Acts {
+
+double AdaptiveGridTrackDensity::getBinCenter(int bin, double binExtent) {
   return bin * binExtent;
 }
 
-int Acts::AdaptiveGridTrackDensity::getBin(float value, float binExtent) {
+int AdaptiveGridTrackDensity::getBin(double value, double binExtent) {
   return static_cast<int>(std::floor(value / binExtent - 0.5) + 1);
 }
 
-typename Acts::AdaptiveGridTrackDensity::DensityMap::const_iterator
-Acts::AdaptiveGridTrackDensity::highestDensityEntry(
+int AdaptiveGridTrackDensity::getTrkGridSize(double sigma, double trkSigmas,
```

Comment:
```suggestion
unsigned int AdaptiveGridTrackDensity::getTrkGridSize(double sigma, double trkSigmas,
```

Reply:
semantically you are right but mixing `int` and `unsigned` is no good